docs: update v19 migration guide to contain instruction about upgrading `@ngrx/signals`

### Which @ngrx/* package(s) are the source of the bug?

signals

### Minimal reproduction of the bug/regression with instructions

When using @ngrx/signals in an Angular application (v18) and using `ng update` to upgrade to version 19, it is reported that @ngrx/store is in need of update.  If you run `ng update @ngrx/store` like the tool suggests, but you don't have this in your package.json, you rightfully get an error saying that @ngrx/store is not a dependency.  You can fix this by running `ng update @ngrx/signals`, but I think if that's the library you are using, the `ng update` command should not tell you to update @ngrx/store.

Also, when you run this update, the migration fails: 

``` log
[error] Error: Cannot find module '@angular-devkit/core'
Require stack:
- C:\Users\jsmith\source\repos\Navajo.Importables\navajo-data\node_modules\@ngrx\signals\schematics-core\utility\find-component.js
- C:\Users\jsmith\source\repos\Navajo.Importables\navajo-data\node_modules\@ngrx\signals\schematics-core\index.js
- C:\Users\jsmith\source\repos\Navajo.Importables\navajo-data\node_modules\@ngrx\signals\migrations\18_0_0-rc_3-protected-state\index.js
- C:\Users\jsmith\AppData\Local\Temp\angular-cli-packages-XXXXXXFNVdtw\node_modules\@angular-devkit\schematics\tools\export-ref.js
- C:\Users\jsmith\AppData\Local\Temp\angular-cli-packages-XXXXXXFNVdtw\node_modules\@angular-devkit\schematics\tools\index.js
- C:\Users\jsmith\AppData\Local\Temp\angular-cli-packages-XXXXXXFNVdtw\node_modules\@angular\cli\src\commands\update\cli.js
- C:\Users\jsmith\AppData\Local\Temp\angular-cli-packages-XXXXXXFNVdtw\node_modules\@angular\cli\src\commands\command-config.js
- C:\Users\jsmith\AppData\Local\Temp\angular-cli-packages-XXXXXXFNVdtw\node_modules\@angular\cli\src\command-builder\command-runner.js
- C:\Users\jsmith\AppData\Local\Temp\angular-cli-packages-XXXXXXFNVdtw\node_modules\@angular\cli\lib\cli\index.js
- C:\Users\jsmith\AppData\Local\Temp\angular-cli-packages-XXXXXXFNVdtw\node_modules\@angular\cli\lib\init.js
    at Module._resolveFilename (node:internal/modules/cjs/loader:1145:15)
    at Module._load (node:internal/modules/cjs/loader:986:27)
    at Module.require (node:internal/modules/cjs/loader:1233:19)
    at require (node:internal/modules/helpers:179:18)
    at Object.<anonymous> (C:\Users\jsmith\source\repos\Navajo.Importables\navajo-data\node_modules\@ngrx\signals\schematics-core\utility\find-component.js:13:14)
    at Module._compile (node:internal/modules/cjs/loader:1358:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1416:10)
    at Module.load (node:internal/modules/cjs/loader:1208:32)
    at Module._load (node:internal/modules/cjs/loader:1024:12)
    at Module.require (node:internal/modules/cjs/loader:1233:19)
```

### Expected behavior

When @ngrx/signals is a dependency, and an update is available, `ng update` should suggest running `ng update @ngrx/signals` instead of `ng update @ngrx/store`

### Versions of NgRx, Angular, Node, affected browser(s) and operating system(s)

NgRx, Angular: 18

### Other information

_No response_

### I would be willing to submit a PR to fix this issue

- [ ] Yes
- [ ] No

--- a/modules/component-store/package.json
+++ b/modules/component-store/package.json
@@ -28,6 +28,7 @@
   "schematics": "./schematics/collection.json",
   "sideEffects": false,
   "ng-update": {
+    "packageGroupName": "@ngrx/store",
     "packageGroup": [
       "@ngrx/store",
       "@ngrx/effects",

--- a/modules/component/package.json
+++ b/modules/component/package.json
@@ -27,6 +27,7 @@
   "schematics": "./schematics/collection.json",
   "sideEffects": false,
   "ng-update": {
+    "packageGroupName": "@ngrx/store",
     "packageGroup": [
       "@ngrx/store",
       "@ngrx/effects",

--- a/modules/data/package.json
+++ b/modules/data/package.json
@@ -29,6 +29,7 @@
   },
   "schematics": "./schematics/collection.json",
   "ng-update": {
+    "packageGroupName": "@ngrx/store",
     "packageGroup": [
       "@ngrx/store",
       "@ngrx/effects",

--- a/modules/effects/package.json
+++ b/modules/effects/package.json
@@ -27,6 +27,7 @@
   },
   "schematics": "./schematics/collection.json",
   "ng-update": {
+    "packageGroupName": "@ngrx/store",
     "packageGroup": [
       "@ngrx/store",
       "@ngrx/effects",

--- a/modules/entity/package.json
+++ b/modules/entity/package.json
@@ -26,6 +26,7 @@
   },
   "schematics": "./schematics/collection.json",
   "ng-update": {
+    "packageGroupName": "@ngrx/store",
     "packageGroup": [
       "@ngrx/store",
       "@ngrx/effects",

--- a/modules/eslint-plugin/package.json
+++ b/modules/eslint-plugin/package.json
@@ -20,6 +20,7 @@
   "homepage": "https://github.com/ngrx/platform#readme",
   "schematics": "./schematics/collection.json",
   "ng-update": {
+    "packageGroupName": "@ngrx/store",
     "packageGroup": [
       "@ngrx/store",
       "@ngrx/effects",

--- a/modules/operators/package.json
+++ b/modules/operators/package.json
@@ -25,6 +25,7 @@
   "schematics": "./schematics/collection.json",
   "sideEffects": false,
   "ng-update": {
+    "packageGroupName": "@ngrx/store",
     "packageGroup": [
       "@ngrx/store",
       "@ngrx/effects",

--- a/modules/router-store/package.json
+++ b/modules/router-store/package.json
@@ -28,6 +28,7 @@
   },
   "schematics": "./schematics/collection.json",
   "ng-update": {
+    "packageGroupName": "@ngrx/store",
     "packageGroup": [
       "@ngrx/store",
       "@ngrx/effects",

--- a/modules/schematics/package.json
+++ b/modules/schematics/package.json
@@ -25,6 +25,7 @@
     "save": "devDependencies"
   },
   "ng-update": {
+    "packageGroupName": "@ngrx/store",
     "packageGroup": [
       "@ngrx/store",
       "@ngrx/effects",

--- a/modules/signals/package.json
+++ b/modules/signals/package.json
@@ -32,6 +32,7 @@
   "schematics": "./schematics/collection.json",
   "sideEffects": false,
   "ng-update": {
+    "packageGroupName": "@ngrx/signals",
     "packageGroup": [
       "@ngrx/store",
       "@ngrx/effects",

--- a/modules/store-devtools/package.json
+++ b/modules/store-devtools/package.json
@@ -26,6 +26,7 @@
   },
   "schematics": "./schematics/collection.json",
   "ng-update": {
+    "packageGroupName": "@ngrx/store",
     "packageGroup": [
       "@ngrx/store",
       "@ngrx/effects",

--- a/modules/store/package.json
+++ b/modules/store/package.json
@@ -26,6 +26,7 @@
   },
   "schematics": "./schematics/collection.json",
   "ng-update": {
+    "packageGroupName": "@ngrx/store",
     "packageGroup": [
       "@ngrx/store",
       "@ngrx/effects",

